doc/ale-terraform.txt

```diff
@@ -114,6 +114,25 @@ g:ale_terraform_tflint_options                 *g:ale_terraform_tflint_options*
   to include '-f json' in your new value.
 
 
+===============================================================================
+tfsec                                                     *ale-terraform-tfsec*
+
+g:ale_terraform_tfsec_executable             *g:ale_terraform_tfsec_executable*
+                                             *b:ale_terraform_tfsec_executable*
+
+  Type: |String|
+  Default: `'tfsec'`
+
+  This variable can be changed to use a different executable for tfsec.
+
+g:ale_terraform_tfsec_options                *g:ale_terraform_tfsec_executable*
+                                             *b:ale_terraform_tfsec_executable*
```

Comment:
Assuming the tag was a typo for `[gb]:ale_terraform_tfsec_options`? This causes `:helptags` to error

Reply:
oh, I'll fix it.

Reply:
@hsanson perhaps it's worth adding something like `:helptags ALL` to the test suite?

Reply:
https://github.com/dense-analysis/ale/issues/4328